prediction_market_agent/utils.py

```diff
@@ -22,6 +22,13 @@ class APIKeys(APIKeysBase):
     SERP_API_KEY: t.Optional[SecretStr] = None
     OPENAI_API_KEY: t.Optional[SecretStr] = None
     TAVILY_API_KEY: t.Optional[SecretStr] = None
+    FARCASTER_PRIVATE_KEY: t.Optional[SecretStr] = None
```

Comment:
I think it makes sense to move these to a separate keys class (like here https://github.com/gnosis/prediction-market-agent/blob/6ae8e2db057be95b6f944924e30ceda9f1f0d8c6/prediction_market_agent/tools/mech/api_keys.py#L9), as these will unlikely be used by other agents.

Reply:
great catch, added